Remove Jackson Smile format, #24155 (#27105)
akka/akka: akka-docs/src/main/paradox/serialization-jackson.md

* `jackson-json` - ordinary text based JSON
* `jackson-cbor` - binary [CBOR data format](https://github.com/FasterXML/jackson-dataformats-binary/tree/master/cbor)

> The binary format is more compact and have slightly better better performance than the JSON format.

Reviewer: ```suggestion
The binary format is more compact, with slightly better performance than the JSON format.
```
